src/docs/010_manual/20_install.adoc

```diff
@@ -127,14 +127,18 @@ container engine and pull the docToolchain image on the first invocation.
 [role='primary']
 --
 
-To install docToolchain in `$HOME/.doctoolchain` execute the following command.
+docToolchain will be installed below the directory specified via the environment variable `DTC_ROOT`, which defaults to `$HOME`.
+To install docToolchain in `$DTC_ROOT/.doctoolchain` execute the following command.
```

Comment:
This is erroneous - check your code above. `DTC_ROOT=${HOME}/.doctoolchain`. I would rephrase the section above with something like:

"Using the wrapper script will install docToolchain into `$HOME/.doctoolchain`."

Your tip already provides all the information a user may need to change docToolchain's installation directory. So why explaining it redundantly.

Reply:
We should move this paragraph just after the header. The Windows wrapper script `dtcw.ps1` has to be changed as well.